docs/_content/ontwikkelaars/algemeen/zgw-api-landschap-grenzen.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "ZGW API-landschap grenzen"
+date: '07-06-2019'
+weight: 50
+---
+
+In het Common Ground API-landschap worden straks alle bronnen netjes ontsloten 
+middels een API. Er is een API die Kadastergegevens ontsluit, een API voor 
+Zaken, Documenten, Personen, Medewerkers, etc.
+
+Op dit moment wordt er hard gewerkt aan een set API's die bij elkaar de
+kern vormen voor Zaakgericht werken, de ZGW API's. Deze bestaan op het moment
+van schrijven uit de Zaken API, Documenten API, Zaaktypen API en Besluiten API.
+Deze maken op hun beurt weer gebruik van de Autorisaties API en de Notificaties 
+API.
+
+## Probleemomschrijving
+
+Een `Zaak` kent typisch een of meer `Betrokkenen` zoals een 
+`Natuurlijk Persoon` of `Vestiging`. Ook gaat een `Zaak` meestal over een of 
+meer `ZaakObjecten` zoals een `Buurt`, `Kunstwerkdeel`, `Ligplaats` of `Wijk`. 
+Allemaal objecten.
+
+Deze objecten "leven" in eigen registraties die middels API's benaderd moeten 
+worden. Aangezien de ZGW API's de eerste set API's is die beschikbaar komt als 
+VNG API standaard, is het API-landschap nog erg beperkt.
+
+De objecten achter de Zaak-relaties `Betrokkenen` en `Zaakobjecten` kunnen 
+daarom niet worden ontsloten middels een API. Er kan dus niet een URL opgegeven
+worden die verwijst naar zo'n object, zoals bijvoorbeeld hieronder is 
+weergegeven:
+
+```http
+GET https://zaken.api.haarlem.nl/v1/zaakobjecten/c9a651  HTTP/1.0
+
+{
+  "url": "https://zaken.api.haarlem.nl/v1/zaakobjecten/c9a651",
+  "zaak": "https://zaken.api.haarlem.nl/v1/zaken/66a38b",
+  "object": "https://objecten.api.haarlem.nl/v1/wijken/3fa723",
+  "relatieomschrijving": "Speelt alleen in deze wijk",
+  "type": "Wijk"
+}
+```
+
+Er wordt bij het `Zaakobject` aan de ene kant prima gerefereerd naar een `Zaak`
+(attribuut `zaak`) maar er kan nog niet verwezen worden naar een `Wijk`
+(attribuut `object`).
+
+**Dit zijn de grenzen van het API-landschap. Een API verwijst naar een object
+dat nog niet ontsloten wordt middels een API.**
+
+## Oplossing
+
+Het RSGB definieert zogenaamde "matchinggegevens" van alle objecten. Een set 
+aan gegevens die nodig zijn om een object te identificeren en op te zoeken. 
+Deze staan het meest helder vermeld in de gepubliceerde XSDs van o.a. [RGBZ] en
+[RSGB], onder de ([onhandige][matching-vs-kern-gegevens]) naam "kerngegevens".
+
+In het eerder getoonde voorbeeld `Wijk` zijn de "matchinggegevens" `wijkCode`, 
+`wijkNaam` en `gem.gemeenteCode`. Deze zijn opgenomen in het bericht, in het 
```

Comment:
De beoogde oplossing is nu om de matchinggegevens op te nemen in het bericht als er niet met een URI kan worden verwezen naar het object